java/src/jmri/jmrit/display/layoutEditor/configurexml/LayoutTurntableViewXml.java

```diff
@@ -48,6 +48,19 @@ public Element store(Object o) {
         element.setAttribute("xcen", "" + coords.getX());
         element.setAttribute("ycen", "" + coords.getY());
         element.setAttribute("turnoutControlled", "" + (turnoutControl ? "yes" : "no"));
+        boolean dispatcherManaged = p.isDispatcherManaged();
+        element.setAttribute("dispatcherManaged", dispatcherManaged ? "yes" : "no");
+        if (dispatcherManaged) {
```

Comment:
You need to move the `element.setAttribute()` call to inside the `if` statement. If `dispatcherManaged` is false, you should not store the attribute.